examples/AspNetCore/Examples.AspNetCore.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Would it be better to extra these out to a single place?

Reply:
Updated